docs/features.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Supported features
+sidebar_label: Features
+---
+
+Margarita is heavily inspired by our <a href="http://kiwi.com" target="_blank">Kiwi.com</a> mobile and web apps. We want to create a full-fledged production ready app for developers, entrepreneurs and companies who want to use our API.
+
+## Supported Features
+
+Margarita is split into 4 parts or functional domains:
+
+- [Search for connections](features#search-for-connections)
+- [Booking](features#booking)
+- [Manage my booking](features#manage-my-booking)
+- [Payments](features#payments)
+
+### Search for connections
+
+Get an access to the database of 15 billion connections virtually interlined by <a href="http://kiwi.com" target="_blank">Kiwi.com</a>.
+
+#### Search form
+
+- The initial screen of the whole app. The user can set here all the options about the connection he is looking for, such as the departure date, the place of origin, etc.
+
+<div>
+<img src="./assets/screens/search.png" alt="Search" title="Search" width="250"/>
+<img src="./assets/screens/search-passenger.png" alt="" title="" width="250"/>
+<img src="./assets/screens/search-trip-type.png" alt="" title="" width="250"/>
+</div>
+
+#### Calendar
+
+- The user can choose whether he wants a single date, a range of dates, or specify how many nights he wants to stay in the selected destination
+
+<div>
+<img src="./assets/screens/range-date-picker.png" alt="" title="" width="250"/>
+<img src="./assets/screens/range-date-picker-nights.png" alt="" title="" width="250"/>
+</div>
+
+#### Place Picker
+
+- We use Tequila for searching and also showing place suggestions in the input field when inputting places.
+
+<div>
+<img src="./assets/screens/place-picker.png" alt="" title="" width="250"/>
+</div>
+
+#### Results
+
+- When the user chooses an option, the card is expanded for showing more information about the given connection.
+
+<div>
+<img src="./assets/screens/results.png" alt="" title="" width="250"/>
+<img src="./assets/screens/result-detail.png" alt="" title="" width="250"/>
+<img src="./assets/screens/result-detail-expanded.png" alt="" title="" width="250"/>
+</div>
+
+### Booking
+
+Use our powerful infrastructure to book the trips and gain your commission - the best one in the industry.
+
+<div>
+<img src="./assets/screens/booking.png" alt="" title="" width="250"/>
+<img src="./assets/screens/booking-passenger.png" alt="" title="" width="250"/>
+</div>
+
+### Manage my booking
```

Comment:
Maby we should mention that this is not working right now so, something like `### Manage my booking (in progress)`